packages/cojson-storage/src/tests/syncManager.test.ts

```diff
@@ -154,11 +155,11 @@ describe("DB sync manager", () => {
         header: true,
         id: coValueIdToLoad,
         sessions: sessionsData.reduce(
-          (acc, sessionRow) => ({
-            ...acc,
-            [sessionRow.sessionID]: sessionRow.lastIdx,
-          }),
-          {},
+          (acc, sessionRow) => {
+            acc[sessionRow.sessionID] = sessionRow.lastIdx;
+            return acc;
+          },
```

Comment:
❤️ 

Thanks for replacing this

Reply:
This is why I'm keen to turn linting on for all the packages